### ISSUE TITLE
Mix watch task fails when adding to/navigating images folder

### ISSUE THREAD

**Issue by lspoor** [JeffreyWay/laravel-mix#905]
So whenever I have `npm run watch` running and create a new folder in the `/resources/assets/images/` folder I get the following error below. This also happens whenever I try to navigate through folders within the same folder which is just crazy. Error I get in terminal is below.

```
Copying resources/assets/images/.DS_Store to /Users/Luke/Desktop/Sites/M3 Website/public_html/images
fs.js:1089
  return binding.unlink(pathModule._makeLong(path));
                 ^

Error: EPERM: operation not permitted, unlink '/Users/Luke/Desktop/Sites/M3 Website/public_html/images'
    at Error (native)
    at Object.fs.unlinkSync (fs.js:1089:18)
    at copyFileSync (/Users/Luke/Desktop/Sites/M3 Website/node_modules/fs-extra/lib/copy-sync/copy-file-sync.js:15:10)
    at Object.copySync (/Users/Luke/Desktop/Sites/M3 Website/node_modules/fs-extra/lib/copy-sync/copy-sync.js:43:5)
    at FileCollection.copyTo (/Users/Luke/Desktop/Sites/M3 Website/node_modules/laravel-mix/src/FileCollection.js:41:12)
    at FSWatcher.chokidar.watch.on.updatedFile (/Users/Luke/Desktop/Sites/M3 Website/node_modules/laravel-mix/src/FileCollection.js:80:47)
    at emitOne (events.js:96:13)
    at FSWatcher.emit (events.js:188:7)
    at FSWatcher.<anonymous> (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/index.js:196:15)
    at FSWatcher._emit (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/index.js:238:5)
    at FSWatcher.<anonymous> (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:204:14)
    at addOrChange (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:210:7)
    at FSWatcher.<anonymous> (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:236:16)
    at filteredListener (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:58:7)
    at /Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:83:11
    at Array.forEach (native)
    at FSEvents.<anonymous> (/Users/Luke/Desktop/Sites/M3 Website/node_modules/chokidar/lib/fsevents-handler.js:82:34)
    at emitThree (events.js:116:13)
    at FSEvents.emit (events.js:194:7)
    at Immediate.<anonymous> (/Users/Luke/Desktop/Sites/M3 Website/node_modules/fsevents/fsevents.js:47:11)
    at runCallback (timers.js:637:20)
    at tryOnImmediate (timers.js:610:5)
    at processImmediate [as _immediateCallback] (timers.js:582:5)

npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "watch"
npm ERR! node v6.8.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! @ watch: `cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ watch script 'cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Luke/Desktop/Sites/M3 Website/npm-debug.log
```

**Comment by JeffreyWay** [comment]
Which version of Mix are you using? You deleted the issue template.

**Comment by lspoor** [comment]
Umm, it came straight from a fresh install of Laravel 5.4, my package.json has this: `"laravel-mix": "0.*",`

**Comment by JeffreyWay** [comment]
Have you altered the default webpack.mix.js file?

**Comment by lspoor** [comment]
Yup, this is my webpack.mix.js file
```
// Require Laravel Mix
// ==================================================
let mix = require('laravel-mix');

// Change public path to public_html
// ==================================================
mix.setPublicPath('public_html/');

// Styles
// ==================================================
mix.sass('resources/assets/scss/app.scss', 'public_html/css')
	.options({
		processCssUrls: false
	});

// Scripts
// ==================================================
mix.js('resources/assets/js/app.js', 'public_html/js')
mix.copy('resources/assets/js/libraries', 'public_html/js/libraries', false);

// Images
// ==================================================
mix.copy('resources/assets/images', 'public_html/images', false);

// Disable Notifications
// ==================================================
mix.disableNotifications();
```

**Comment by jicao** [comment]
Hello,

Did you manage to solve the problem ? I have the same trouble each X time I make a change when i "npm run watch"... I'm so crazy about it... SO damn enoying !

```
[Browsersync] Watching files...
 WAIT  Compiling...17:03:13

 95% emitting DONE  Compiled successfully in 1030ms17:03:14

       Asset    Size  Chunks             Chunk Names
/css/app.css  230 kB    0, 0  [emitted]  /js/app, /js/app
 + 2 hidden assets
[Browsersync] File event [change] : public\css\app.css
 WAIT  Compiling...17:03:21

 95% emitting DONE  Compiled successfully in 753ms17:03:22

fs.js:1089
  return binding.unlink(pathModule._makeLong(path));
                 ^

Error: EBUSY: resource busy or locked, unlink 'C:\Users\MYPATH\laravel-homestead\Laravel\public\img\partnerscarousel\Logo.svg'
    at Error (native)
    at Object.fs.unlinkSync (fs.js:1089:18)
    at copyFileSync (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-file-sync.js:15:10)
    at copySync (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-sync.js:43:5)
    at contents.forEach.content (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-sync.js:54:7)
    at Array.forEach (native)
    at copySync (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-sync.js:51:14)
    at contents.forEach.content (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-sync.js:54:7)
    at Array.forEach (native)
    at Object.copySync (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\fs-extra\lib\copy-sync\copy-sync.js:51:14)
    at File.copyTo (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\File.js:190:12)
    at FileCollection.copyTo (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\FileCollection.js:82:24)
    at src.forEach.file (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\FileCollection.js:76:38)
    at Array.forEach (native)
    at FileCollection.copyTo (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\FileCollection.js:76:17)
    at CopyFilesTask.run (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\tasks\CopyFilesTask.js:14:20)
    at CustomTasksPlugin.runTask (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\webpackPlugins\CustomTasksPlugin.js:33:14)
    at Mix.tasks.forEach.task (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\webpackPlugins\CustomTasksPlugin.js:9:44)
    at Array.forEach (native)
    at Compiler.compiler.plugin.stats (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\laravel-mix\src\webpackPlugins\CustomTasksPlugin.js:9:23)
    at Compiler.applyPlugins (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\tapable\lib\Tapable.js:61:14)
    at Watching._done (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:105:17)
    at compiler.emitRecords.err (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:79:19)
    at Compiler.emitRecords (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:371:38)
    at compiler.emitAssets.err (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:62:20)
    at applyPluginsAsyncSeries1.err (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:364:12)
    at next (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\tapable\lib\Tapable.js:218:11)
    at Compiler.compiler.plugin (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\performance\SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\tapable\lib\Tapable.js:222:13)
    at Compiler.afterEmit (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:361:9)
    at require.forEach.err (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\webpack\lib\Compiler.js:350:15)
    at C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\async\dist\async.js:473:16
    at iteratorCallback (C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\async\dist\async.js:1050:13)
    at C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\async\dist\async.js:958:16
    at C:\Users\MYPATH\laravel-homestead\Laravel\node_modules\graceful-fs\graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:123:15)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ watch: `cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\myuser\AppData\Roaming\npm-cache\_logs\2018-06-27T15_03_22_936Z-debug.log

```

Thanks

**Comment by Tarasovych** [comment]
Probably [fixed in `webpack-cli 3.3.3`](https://github.com/webpack/webpack-cli/issues/870)
